package/src/frontend/media/media.js

```diff
@@ -32,7 +45,32 @@ export const media = {
     if (player) {
       this.playerPool.unAllocatePlayer(player);
     }
-  }
+  },
+  updateAtmo: function (config) {
+    if (config.sources) {
+      this.sourceIDMap.mapSources(config.atmoAudioFileId, config.sources);
+    }
+    this.atmo.atmoSourceId = config.atmoAudioFileId;
+    this.atmo.update();
+  },
 };
 
+media.Atmo = Atmo;
+media.atmo = new Atmo({
+  pool: media.sourceIDMap,
+  backgroundMedia: media,
+  multiPlayer: {
+    loop: true,
+    fadeDuration: 500,
+    crossFade: true,
+    playFromBeginning: false,
+    rewindOnChange: true,
+    pauseInBackground: true
+  }
+});
+
+browser.detectFeatures().then(()=>{
+  media.atmo.disabled =  browser.has('mobile platform');
+});
+
```

Comment:
The most immediate problem, which is causing the specs to fail, is that we do not want to call `detectFeatures` here. This is easy to solve since there is no need to disable atmo on mobile devices. This was only necessary since we had no way to reliably auto play atmo audios. With media pool we now do. So this can just be removed.

The second problem is that we do not want to create these `Atmo` objects in Paged entries since they (at least for now) have their own atmo implementation. So we will have to move away from the `atmo` singleton. As noted above, the `Atmo` spec can be decoupled from the singleton by working with a local `Atmo` instance.

Finally, I'd suggest to separate atmo and media concerns more strictly. Before, the media module mainly had two responsibilities:

* Provide players 
* Handle unmuting

Now we've added

* Maintain a map of sources by id
* Provide atmo functionality

The only overlap between these two pieces of functionality is that `getPlayer` needs to access `atmo` to create hooks.  If we can remove this dependency, atmo can be extracted from `media` and live next to it in `src/frontend/Atmo.js`. Here's one way to do it:

* Introduce `AtmoProvider` component in `pageflow-scrolled/frontend` that
  * creates a SourceIdMap object
  * creates a MultiPlayer passing the SourceIdMap as pool
  * creates an `Atmo` object passing MultiPlayer
  * places a function just like `media.updateAtmo` on the context which stores the sources in the SourceIdMap and updates the atmo with the given id
  * places a function that calls `atmo.createMediaPlayerHooks` on the React context. 

* Extract the `onActivate` code in `Section` into a `AtmoAudio` component that is rendered in `Section` inside the `SectionLifecycleProvider`. It can then look up the `updateAtmo` function from the Reat context and the audio file via `useFile` and use the `onActivate`/`onDeactivate` options of the `useSectionLifecycle` to update the atmo. This also fixes the problem with the current code that the atmo is never faded out when a section without atmo audio is reached. The component can simply render `null`.

* In `PlayerContainer`, get the `atmo.createMediaPlayerHooks` function from the context and call it to obtain media player hooks to pass to `getPlayer`. That way `getPlayer` does not need to know about `atmoDuringPlayback` - only about hooks.

* Everything related to SourceIdMap and atmo can then be removed from `media`.

* In Scrolled, this could be tested with an integration/feature spec ([example](https://github.com/codevise/pageflow/blob/master/entry_types/scrolled/package/spec/frontend/features/changeSectionMessage-spec.js)) that uses [`fake media`](https://github.com/codevise/pageflow/blob/master/entry_types/scrolled/package/spec/support/fakeMedia.js) to stub out the actual media player code and test that `media.getPlayer` is called with the expected sources when a section when the user scrolls to a section with an atmo audio file.